libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -115,6 +115,14 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_FILE_MB_FREE",  7, AP_Logger, _params.min_MB_free, 500),
 
+    // @Param: _FILE_RATEMAX
+    // @DisplayName: Maximum logging rate
+    // @Description: This sets the maximum rate that streaming log messages will be logged to the file backend. A value of zero means
+    // @Units: Hz
+    // @Range: 0 1000
+    // @User: Standard
+    AP_GROUPINFO("_FILE_RATEMAX",  8, AP_Logger, _params.file_ratemax, 0),
```

Comment:
This should be done for the flash backend as well (it would be useful), and should be called something that means its not just for file.

Reply:
yes, the plan is to have 3 separate parameters, so we can have different rates for mavlink and file/block